mu/logic.py

```diff
@@ -631,7 +663,8 @@ def quit(self, *args, **kwargs):
                 paths.append(widget.path)
         session = {
             'theme': self.theme,
-            'paths': paths
+            'paths': paths,
+            'python_dir': get_python_dir()
```

Comment:
Could we use a different name for this? if I am reading the settings file and I see "python_dir" I would assume it would need to point to the python interpreter.
If we do change this, then I guess `get_python_dir()` should also be renamed as well.